draft-ietf-quic-tls.md

```diff
@@ -1576,12 +1576,16 @@ remove packet protection, and results in all packets with an invalid Key Phase
 bit being rejected.
 
 The process of creating new packet protection keys for receiving packets could
-reveal that a key update has occurred.  An endpoint MAY perform this process as
-part of packet processing, but this creates a timing signal that can be used by
-an attacker to learn when key updates happen and thus the value of the Key Phase
-bit in certain packets.  Endpoints MAY instead defer the creation of the next
-set of receive packet protection keys until some time after a key update
-completes, up to three times the PTO; see {{old-keys-recv}}.
+reveal that a key update has occurred. An endpoint MAY generate new keys as
+part of packet processing, but this creates a timing signal that could be used
+by an attacker to learn when key updates happen and thus leak the value of the
+Key Phase bit.
+
+Endpoints are generally expected to have current and next receive packet
+protection keys available. For a short period after a key update completes, up
+to three times the PTO, endpoints MAY defer generation of the next set of
```

Comment:
```suggestion
to the PTO, endpoints MAY defer generation of the next set of
```

@janaiyengar, @kazuho can you check this?  I think that this is correct and matches the recommendation that is cited.  3PTO is the period over which we ask endpoints to refrain from key updates.

Reply:
One PTO is also what is mentioned at L1638-L1640.

Reply:
I think "defer generation of the subsequent set" might be good (and be consistent with L1640).  (Not phrasing as a suggestion since the two suggestions would conflict.)

Reply:
:+1: I think that the suggested change is correct.

Our recommendations are that the _receiver_ keeps the old keys for one PTO, the _sender_ delays the next key update for at least three PTO after it receives an ACK for the current key update. This paragraph is talking about the receiver, so it should be one PTO.